source/getting-started/index.markdown

```diff
@@ -1,86 +1,14 @@
 ---
-title: "Install Home Assistant"
-description: "Getting started: How to install Home Assistant."
+title: "Getting Started"
+description: "Getting started with Home Assistant"
+hide_github_edit: true
```

Comment:
```suggestion
feedback: false
```

Reply:
This actually does not make sense to hide here, we want feeedback :smile: 
```suggestion
```

Reply:
Even better :)